components/home/actions.js

```diff
@@ -0,0 +1,26 @@
+import { UPDATE_REACTION, REQUEST_UPLOAD_LIST } from './actionTypes';
+
+/**
+ * Triggers request to react on a post
+ *
+ * @function
+ * @return {Object} The {@link actionTypes.REQUEST_UPLOAD_LIST REQUEST_UPLOAD_LIST}
+ * action.
+ */
+export const updateReaction = (id, reaction) => ({
```

Comment:
make this action follow the same pattern as `requestUploadList`

Reply:
OK.